server/src/lsp-server.spec.ts

```diff
@@ -456,6 +457,292 @@ describe('signatureHelp', () => {
+                                                    line: 1
+                                                },
+                                                start: {
+                                                    character: 24,
+                                                    line: 1
+                                                }
+                                            }
+                                        },
+                                        {
+                                            newText: '',
+                                            range: {
+                                                end: {
+                                                    character: 16,
+                                                    line: 2
+                                                },
+                                                start: {
+                                                    character: 8,
+                                                    line: 2
+                                                }
+                                            }
+                                        }
+                                    ],
+                                    textDocument: {
+                                        uri: 'file://ROOT/server/test-data/bar.ts',
+                                        version: 1
+                                    }
+                                }
+                            ]
+                        }
+                    ],
+                    command: '_typescript.applyWorkspaceEdit',
+                    title: "Remove unused declaration for: 'bar'"
+                },
+                kind: 'quickfix',
+                title: "Remove unused declaration for: 'bar'"
+            },
+            {
+                command: {
+                    arguments: [
+                        {
+                            documentChanges: [
+                                {
+                                    edits: [
+                                        {
+                                            newText: '_bar',
+                                            range: {
+                                                end: {
+                                                    character: 27,
+                                                    line: 1
+                                                },
+                                                start: {
+                                                    character: 24,
+                                                    line: 1
+                                                }
+                                            }
+                                        }
+                                    ],
+                                    textDocument: {
+                                        uri: 'file://ROOT/server/test-data/bar.ts',
```

Comment:
Use `uri('bar.ts')` where you expect URI and `filePath('bar.ts')` where you expect file path and remove those hacky replaces.

Reply:
great!